rackcity/views/rackcity_utils.py

```diff
@@ -311,3 +318,13 @@ def get_filter_arguments(data):
                 )
 
     return filter_args
+
+
+def close_old_connections_decorator(func):
+    @functools.wraps(func)
+    def wrapper_decorator(*args, **kwargs):
+        close_old_connections()
+        value = func(*args, **kwargs)
+        close_old_connections()
+        return value
+    return wrapper_decorator
```

Comment:
This is the start of an experimental fix for the database connections issue - it fixed itself before I could test, but was going to leave this here in case it comes up again and someone wants to try commenting out the decorators  I added to asset views

Reply:
If this is the route we would want to go (I don't have high conviction that it is), I think it would ultimately make sense to roll this decorator in the @api_view decorator so we don't have to repeat it every time